Gems/AtomTressFX/Code/Passes/HairParentPass.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RHI/DrawListTagRegistry.h>
+#include <Atom/RHI/RHISystemInterface.h>
+#include <Atom/RPI.Public/RenderPipeline.h>
+#include <Atom/RPI.Reflect/Pass/RasterPassData.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        HairParentPass::HairParentPass(const RPI::PassDescriptor& descriptor)
+            : Base(descriptor)
+        {
+        }
+
+        HairParentPass::~HairParentPass()
+        {
+        }
+
+        RPI::Ptr<HairParentPass> HairParentPass::Create(const RPI::PassDescriptor& descriptor)
+        {
+            return aznew HairParentPass(descriptor);
+        }
+
+        void HairParentPass::UpdateChildren()
+        {
+            if (!m_updateChildren)
+            {
+                return;
+            }
+            m_updateChildren = false;
+        }
```

Comment:
This function can be deleted, no? Doesn't seem to do anything, and m_updateChildren isn't used anywhere else

Reply:
Correct,
I want to keep it for a while as it is the foundation of adding control over passes via code and in this case I might want to allow enabling / disabling different pass chains (using short method vs' full buffers as now), enabling collisions, etc..
I'll add a remark exactly about this in code to clarify why it's there.
Let me know if I can close this remark